static.config.js

```diff
@@ -742,9 +741,21 @@ const makeAllPages = async (langCode, incrementalPageId) => {
     }),
   );
 
+  // polyfill
```

Comment:
Do we need this polyfill? It looks like .flat() should be supported on nodejs 11 and later

Reply:
okay, ill remove polyfill